ytfzf

```diff
@@ -1594,6 +1597,7 @@ parse_opt () {
 
 		l|loop)
 			enable_loop=${optarg:-1}
+			enable_persistent_key_binds=1
```

Comment:
That should be `enable_persistent_key_binds=${optarg:-1}`

Reply:
yes